changelog.d/20240627_181028_chris_mep_config_user_runtime.rst

```diff
@@ -0,0 +1,7 @@
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- MEP config templates now receive information about the runtime environment used to
+  submit tasks, such as Python environment and Globus Compute SDK version, via the
```

Comment:
The MEP is still new enough (I'm guessing for the next ~year), that I suggest that we either spell it out explicitly (i.e., _Multi-user endpoint configuration templates ..._), set up an acronym (e.g., ``:abbr:`MEP (multi-user compute endpoint)` ``), or link to the [MEP documentation page](https://globus-compute.readthedocs.io/en/latest/endpoints/multi_user.html).